spec/system/candidate_interface/submitting/candidate_submitting_application_with_interruption_pages_spec.rb

```diff
@@ -56,11 +56,74 @@
     then_the_viewed_enic_interruption_page_cookie_to_be_set
   end
 
+  scenario 'Candidate submits an teacher degree apprenticeship course having a personal statement less than 500 words and a degree', :js, time: mid_cycle do
+    given_i_am_signed_in
+    when_i_have_completed_my_undergraduate_application_and_have_added_primary_as_a_course_choice_with_short_personal_statement_and_a_degree
+    and_i_continue_with_my_application
+
+    when_i_save_as_draft
+    and_i_am_redirected_to_the_application_dashboard
+    and_my_application_is_still_unsubmitted
+    and_i_continue_with_my_application
+
+    when_i_click_to_review_my_application
+    then_i_see_a_interruption_page_for_personal_statement
+    when_i_continue_without_editing
+    then_i_see_a_interruption_page_for_degree_warning
+    when_i_click_to_continue_and_apply_for_the_course
+    then_i_see_the_review_and_submit_page
+  end
+
+  scenario 'Candidate submits an teacher degree apprenticeship course having a degree with long personal statement and with ENIC', :js, time: mid_cycle do
+    given_i_am_signed_in
+    when_i_have_completed_my_undergraduate_application_and_have_added_primary_as_a_course_choice
+    and_i_continue_with_my_application
+
+    when_i_save_as_draft
+    and_i_am_redirected_to_the_application_dashboard
+    and_my_application_is_still_unsubmitted
+    and_i_continue_with_my_application
+
+    when_i_click_to_review_my_application
+    then_i_see_a_interruption_page_for_degree_warning
+    when_i_click_to_continue_and_apply_for_the_course
+    then_i_see_the_review_and_submit_page
+  end
+
   def when_i_have_completed_my_application_and_have_added_primary_as_a_course_choice_with_not_needed_qualification
     given_i_have_a_primary_course_choice(application_form_completed: true, personal_statement_words: 4)
     add_not_needed_qualification
   end
 
+  def when_i_have_completed_my_undergraduate_application_and_have_added_primary_as_a_course_choice_with_short_personal_statement_and_a_degree
```

Comment:
I think this should be split on the `and` into two methods so it's easier to read

Reply:
Thanks for spot this. I moved the `and` to the scenario.

Reply:
The method still has an `and` in it

```ruby
  scenario 'asdf' do
    when_i_have_completed_my_undergraduate_application
    and_have_added_primary_as_a_course_choice
  end

  def and_have_added_primary_as_a_course_choice
    given_i_have_a_primary_course_choice(application_form_completed: true, personal_statement_words: 500)
  end

 def when_i_have_completed_my_undergraduate_application
    and_i_have_a_degree
    and_course_choice_is_undergraduate
  end
```

Reply:
I changed now